tests/testapp/tests/test_app_api.py

```diff
@@ -15,6 +16,12 @@ class CommentAppAPITests(CommentTestCase):
     def testGetCommentApp(self):
         self.assertEqual(django_comments.get_comment_app(), django_comments)
 
+    @modify_settings(INSTALLED_APPS={'remove': 'django_comments'})
```

Comment:
@claudep, sorry my PR was a bit rushed. I thought this test still had value if rewritten a bit.